CHANGELOG.rst

```diff
@@ -6,37 +6,63 @@ CHANGELOG
+* Add support for AWS Batch scheduler in China partition and in ``eu-north-1``.
+* Revamped ``pcluster configure`` command which now supports automated networking configuration.
+* Add support for NICE DCV on Centos 7. You can now connect graphically to the desktop with DCV using
+  ``pcluster dcv connect``.
+* Add support for new EFA supported instances: ``c5n.metal``, ``m5dn.24xlarge``, ``m5n.24xlarge``, ``r5dn.24xlarge``,
+  ``r5n.24xlarge``
+* Add support for scheduling with GPU options in Slurm. Currently supports the following GPU-related options: ``—G/——gpus,
+  ——gpus-per-task, ——gpus-per-node, ——gres=gpu, ——cpus-per-gpu``.
+  Integrated GPU requirements into scaling logic, cluster will scale automatically to satisfy GPU/CPU requirements
+  for pending jobs. When submitting GPU jobs, CPU/node/task information is not required but preferred in order to
+  avoid ambiguity. If only GPU requirements are specified, cluster will scale up to the minimum number of nodes
+  required to satisfy all GPU requirements.
+* Add new cluster configuration option to automatically disable Hyperthreading (``disable_hyperthreading = true``)
+* Install Intel Parallel Studio 2019.5 Runtime in Centos 7 AMI and share /opt/intel over NFS.
+* Additional EC2 IAM Policies can now be added to the role ParallelCluster automatically creates for cluster nodes by
+  simply specifying ``additional_iam_policies`` in the cluster config.
 
 **CHANGES**
 
 * Ubuntu 14.04 is no longer supported
+* Upgrade Intel MPI to version U5.
+* Upgrade EFA Installer to version 1.6.2, this also upgrades Open MPI to 4.0.2.
+* Upgrade NVIDIA driver to Tesla version 418.87.
+* Upgrade CUDA library to version 10.1.
+* Upgrade Slurm to version 19.05.3-2.
+* Install EFA in China AMIs.
 * Increase default EBS volume size from 17GB to 25GB
-* Search for new available version only at ``pcluster create`` action
-* EFA 1.6.2 Installer - this also updates Open MPI to 4.0.2
-* Updated Slurm to version 19.05
-* FSx Lustre now mounts with flags ``defaults,_netdev,flock`` and others
-* Updated Intel MPI to version U5
+* FSx Lustre now supports new storage_capacity options 1,200 and 2,400 GiB
+* Enable ``flock user_xattr noatime`` Lustre mount options by default everywhere and
+  ``x-systemd.automount x-systemd.requires=lnet.service`` for systemd based systems.
+* Increase the number of hosts that can be processed by scaling daemons in a single batch from 50 to 200. This
+  improves the scaling time especially with increased ASG launch rates.
+* Change default sshd config in order to disable X11 forwarding and allow only secure ciphers.
+  significantly increases scaling speed when ASG launch rate is raised.
+* Increase faulty node termination timeout from 1 minute to 5 in order to give some additional time to the scheduler
+  to recover when under heavy load.
 * Add ``--cluster-template``, ``--vpc-id``, ``-no-public-ip`` and ``--subnet-id`` parameters to ``pcluster createami`` command
```

Comment:
Extended ``pcluster createami`` command to specify the VPC and network settings for building AMIs.